### PR TITLE
docs: update docs for `client_attrs`

### PR DESCRIPTION
**Pull request by lafirest**
https://github.com/emqx/emqx/pull/13573
https://github.com/emqx/emqx/pull/13604

### CODE REVIEW

**Comment by lafirest** [comment]
@Meggielqk 
seems we need to merge release-5.7 into release-5.8?

**Comment by Meggielqk** [comment]
I have moved the updates to #2605 and this PR can be closed.

File: en_US/data-integration/rule-sql-events-and-fields.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Refer to the table below for fields that can be selected from the received MQTT
 | `timestamp`           | Timestamp (unit: ms)                                |
 | `publish_received_at` | Time when PUBLISH message reaches EMQX (unit: ms)   |
 | `node`                | Node where the event is triggered<!--tech review--> |
+| `client_attrs`        | [client attributes](../client-attributes/client-attributes.md) |
```
Comment (Meggielqk) [inline en_US/data-integration/rule-sql-events-and-fields.md]:
```suggestion
| `client_attrs`        | [Client attributes](../client-attributes/client-attributes.md) |
```

File: en_US/data-integration/rule-sql-events-and-fields.md [hunk 2]
```diff
@@ -323,6 +324,7 @@ Refer to the table below for fields that can be selected from the received MQTT
 | `conn_props`      | CONNECT Properties (MQTT 5.0 clients only)                   |
 | `timestamp`       | Event trigger time (unit: ms)                                |
 | `node`            | EMQX node where the event is triggered                       |
+| `client_attrs`        | [client attributes](../client-attributes/client-attributes.md) |
```
Comment (Meggielqk) [inline en_US/data-integration/rule-sql-events-and-fields.md]:
```suggestion
| `client_attrs`        | [Client attributes](../client-attributes/client-attributes.md) |
```

File: en_US/data-integration/rule-sql-events-and-fields.md [hunk 3]
```diff
@@ -365,6 +367,7 @@ Output:
 | `disconn_props`   | DISCONNECT Properties (MQTT 5.0 clients only)                |
 | `timestamp`       | Event trigger time (unit: ms)                                |
 | `node`            | EMQX node where the event is triggered                       |
+| `client_attrs`        | [client attributes](../client-attributes/client-attributes.md) |
```
Comment (Meggielqk) [inline en_US/data-integration/rule-sql-events-and-fields.md]:
```suggestion
| `client_attrs`        | [Client attributes](../client-attributes/client-attributes.md) |
```

File: en_US/data-integration/rule-sql-events-and-fields.md [hunk 4]
```diff
@@ -507,6 +510,7 @@ Refer to the table below for fields that can be extracted.
 | `authz_source `  | The authorization source |
 | `timestamp` | Timestamp (unit: ms)                                         |
 | `node`      | EMQX node where the event is triggered.                      |
+| `client_attrs`        | [client attributes](../client-attributes/client-attributes.md) |
```
Comment (Meggielqk) [inline en_US/data-integration/rule-sql-events-and-fields.md]:
```suggestion
| `client_attrs`        | [Client attributes](../client-attributes/client-attributes.md) |
```

File: en_US/data-integration/rule-sql-events-and-fields.md [hunk 5]
```diff
@@ -547,6 +551,7 @@ Refer to the table below for fields that can be extracted.
 | `reason_code`     | Authentication result                                  |
 | `is_superuser`    | Whether this client is a super user                    |
 | `is_anonymous`    | Whether this client is a anonymous user                |
+| `client_attrs`        | [client attributes](../client-attributes/client-attributes.md) |
```
Comment (Meggielqk) [inline en_US/data-integration/rule-sql-events-and-fields.md]:
```suggestion
| `client_attrs`        | [Client attributes](../client-attributes/client-attributes.md) |
```

File: en_US/data-integration/rule-sql-events-and-fields.md [hunk 6]
```diff
@@ -587,6 +592,7 @@ Refer to the table below for fields that can be extracted.
 | `sub_props` | SUBSCRIBE Properties (MQTT 5.0 client only) |
 | `timestamp` | Event trigger time (unit: ms)               |
 | `node`      | EMQX node where the event is triggered      |
+| `client_attrs`        | [client attributes](../client-attributes/client-attributes.md) |
```
Comment (Meggielqk) [inline en_US/data-integration/rule-sql-events-and-fields.md]:
```suggestion
| `client_attrs`        | [Client attributes](../client-attributes/client-attributes.md) |
```

File: en_US/data-integration/rule-sql-events-and-fields.md [hunk 7]
```diff
@@ -624,6 +630,7 @@ Refer to the table below for fields that can be extracted.
 | `unsub_props` | UNSUBSCRIBE Properties (MQTT 5.0 clients only) |
 | `timestamp`   | Event trigger time (unit: ms)                  |
 | `node`        | EMQX node where the event is triggered         |
+| `client_attrs`        | [client attributes](../client-attributes/client-attributes.md) |
```
Comment (Meggielqk) [inline en_US/data-integration/rule-sql-events-and-fields.md]:
```suggestion
| `client_attrs`        | [Client attributes](../client-attributes/client-attributes.md) |
```

File: zh_CN/data-integration/rule-sql-events-and-fields.md [hunk 8]
```diff
@@ -510,6 +514,7 @@ FROM
 | `reason_code`     | 认证结果                           |
 | `is_superuser`    | 是否是超级用户                      |
 | `is_anonymous`    | 是否是匿名用户                      |
+| client_attrs        | [客户端属性](../client-attributes/client-attributes.md) |
```
Comment (Meggielqk) [inline zh_CN/data-integration/rule-sql-events-and-fields.md]:
```suggestion
| `client_attrs`        | [客户端属性](../client-attributes/client-attributes.md) |
```

**Review by Meggielqk — APPROVED** [review]

**Pull request merged**